package.json

```diff
@@ -13,13 +13,16 @@
   "keywords": [
     "babel"
   ],
-  "peerDependencies": {
```

Comment:
@hax 
Because we can use it without ``babel``. And another reason is, more and more new projects are based on babel 6. I think that we can't interfere other's choice.